web-ui/src/main/resources/catalog/components/common/map/mapService.js

```diff
@@ -785,7 +808,7 @@
               advanced: options.advanced,
               minResolution: options.minResolution,
               maxResolution: options.maxResolution,
-              cextent: options.extent,
```

Comment:
`cextent` is only used by GeoNetwork internally, but OpenLayers does not expect this property. Therefore, we replace it by `extent` instead.